common-files/utils/httputils.mjs

```diff
@@ -234,18 +236,59 @@ const responseLogger = (req, res, next) => {
   }
 };
 
+const isEndpointWhitelisted = req => {
+  let result = false;
+  req.app.get(ENDPOINTS_WHITELISTED).forEach(e => {
+    if (!result) {
+      result = req.url.match(e) != null;
+    }
+  });
+
+  return result;
+};
+
+const authenticationHandler = (req, res, next) => {
+  if (
+    req.url === '/healthcheck' ||
```

Comment:
Same question here, what is someone makes a requests to `/healthcheck/`. This should be handled by `new URL` or `req.url.startsWith('/healthcheck')`.